media/js/firefox/all/all-downloads-unified-init.es6.js

```diff
@@ -25,11 +25,19 @@ import MzpModal from '@mozilla-protocol/core/protocol/js/modal';
                 className: 'help-modal'
             });
 
+            // UA
             window.dataLayer.push({
                 event: 'in-page-interaction',
                 eAction: 'link click',
                 eLabel: eventLabel
             });
+            // GA4
+            window.dataLayer.push({
+                event: 'widget_action',
+                type: 'modal',
+                action: 'open',
+                name: eventLabel
```

Comment:
eventLabel could be one of: 'Get Browser Help', 'Get Installer Help' but neither is displayed to the user. I waffled on if these should be the name or the label.